src/validation.cpp

```diff
@@ -1129,11 +1129,15 @@ static std::pair<CAmount, CAmount> GetBlockSubsidyHelper(int nPrevBits, int nPre
         dDiff = ConvertBitsToDouble(nPrevBits);
     }
 
-    if (fV20Active) {
-        // Starting from V20 Activation, subsidybase should be stable.
-        // Currently, nSubsidyBase calculate relies on difficulty.
-        // Once Platform is live, it must constantly get blocks difficulty in order to calculate platformReward.
-        // This can not be continued so we set the nSubsidyBase to a fixed value.
+    bool isDevnet = Params().NetworkIDString() == CBaseChainParams::DEVNET;
+    bool force_fixed_base_subsidy = fV20Active || (isDevnet && nPrevHeight >= consensusParams.nHighSubsidyBlocks);
```

Comment:
```suggestion
    bool force_fixed_base_subsidy = fV20Active || (nPrevHeight >= consensusParams.nHighSubsidyBlocks);
```

`nHighSubsidyBlocks` equals 0 to all nets but devnet; so, this condition is quite useless.

Reply:
this won't work on non-devnets

Reply:
ah yes, for `nPrevheight >= consensusParams.nHighSubsidyBlocks` we indeed need a condition `isDevnet`